test-integ/peering_commontopo/main_test.go

```diff
@@ -2,33 +2,25 @@ package peering
 
 import (
 	"flag"
+	"os"
 	"testing"
 )
 
-var FlagNoReuseCommonTopo *bool = flag.Bool("no-reuse-common-topo", false,
-	"run tests that can use the common topo in separate instances")
-
 type commonTopoSuite interface {
 	testName() string
 	setup(*testing.T, *commonTopo)
 	test(*testing.T, *commonTopo)
 }
 
-// Tests compatible with commonTopo should implement commonTopoSuite and be added
-// to the commonTopoSuites slice inside.
-//
-// func setup is executed in serial with others. They should ensure any resources
-// added to ct.Cfg et all do not collide with other resources (e.g. with a prefix)
-//
-// func test is executed in parallel with others in a subtest. (test() should not
-// call t.Parallel() itself.)
-func TestAllCommonTopoSuites(t *testing.T) {
-	if *FlagNoReuseCommonTopo {
-		t.Skip("NoReuseCommonTopo set")
-	}
-	if allowParallelCommonTopo {
-		t.Parallel()
+func CommonTopologyTest(t *testing.T) {
+	t.Helper()
+	if os.Getenv("COMMON_TOPOLOGY") != "true" {
+		t.Skip(`Skipping: set COMMON_TOPOLOGY to "true" run this test`)
 	}
+}
+
+func TestAllCommonTopoSuites(t *testing.T) {
```

Comment:
Come to think of it, this name is now not fully accurate. Maybe something like `TestSuitesOnSharedTopo`